src/main/java/seedu/storage/LoginStorage.java

```diff
@@ -0,0 +1,40 @@
+package seedu.storage;
+
+import seedu.classes.Ui;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class LoginStorage {
+    private static final String PASSWORD_FILE_PATH = "./password.txt";
+
+    public static int load(Ui ui) {
+        try {
+            File f = new File(PASSWORD_FILE_PATH);
+            boolean isFileCreated = f.exists();
```

Comment:
```suggestion
            boolean isFileCreated = new File(PASSWORD_FILE_PATH).createNewFile();
```

Reply:
I find this to be not as readable, as f is still used later on to initialise Scanner